content/en/releases/version-skew-policy.md

```diff
@@ -103,6 +103,13 @@ Example:
 The supported version skew between components has implications on the order in which components must be upgraded.
 This section describes the order in which components must be upgraded to transition an existing cluster from version **{{< skew currentVersionAddMinor -1 >}}** to version **{{< skew currentVersion >}}**.
 
+When preparing to upgrade, the Kubernetes project recommends that your components
```

Comment:
while I think this is a good recommendation, I also think it's a little confusing to mix into a page that draws pretty hard supported/unsupported lines and uses "must" language. Placing this here would make me ask questions like "if I don't do this, does this mean I'm doing unsupported upgrades?"

Reply:
What's the upsell for doing this? If there's a good plus to go from latest patch to latest patch maybe we can add that "To benefit from blah, the Kubernetes project recommends..."

Reply:
I guess just the typical logic that the latest patch version is most likely to include fixes for any regressions or known bugs that could impact the upgrade, but that's not unique to upgrading, it's just generally true for bug/regression fixes.

Reply:
I think it's ok to include a recommendation on this page as a "You must do all these things, and you can also do this thing to avoid unfixed bugs that might mess with your upgrade".

Reply:
We should just make it very clear that it's a suggestion, and fine to ignore